finality-aleph/src/network/clique/service.rs

```diff
@@ -133,7 +135,7 @@ where
         let dialer = self.dialer.clone();
         let next_to_interface = self.next_to_interface.clone();
         self.spawn_handle
-            .spawn("aleph/validator_network_outgoing", None, async move {
+            .spawn("aleph/clique_network_outgoing", None, async move {
```

Comment:
Since `LOG_TARGET` inspired me, that maybe almost every string should be const at the begining of the file? Only exception I can think of are error/expect messages, but probably there are more

Reply:
If these are used only once (and these are, I think you missed the "outgoing"/"incoming" distinction) it's not as important. I think we should do the `LOG_TARGET` thing with log targets in general though, but this is definitely not the PR for that.